index.html

```diff
@@ -4,9 +4,152 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Youtube page by Adorer</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-    
+    <nav class = "flex-div">
+        <div class="nav-left flex-div">
+            <img src="images/menu.png" alt="menu-icon" class="menu-icon">
+            <img src="images/logo.png" alt="youtube icon" class="logo-icon">
+        </div>
+
+        <div class="nav-center flex-div">
+            <div class="search-box flex-div">
+                <input type="text" placeholder="Search">
+                <img src="images/search.png" alt="search-icon">
+             </div>
+             <img src="images/voice-search.png" alt="voice-icon" class = "voice-icon">
+        </div>
+
+        <div class="nav-right flex-div" >
+            <img src="images/upload.png" alt="upload-icon">
+            <img src="images/more.png" alt="images-icon">
+            <img src="images/notification.png" alt="notification-icon">
+            <img src="images/Jack.png" alt="menu icon" class = "user-icon">
+        </div>
```

Comment:
How about using unordered list <ul> here. It should look better right?And don't forget to add an <a> tag to make it clickable

Reply:
- [ ] Please always keep in mind while doing code review. kindly communicate politely. suggest instead of giving order this way the student feels comfortable and doesn't not frustrate. and also try to give `WHY` you requested changes adding more details you let the student understand better that what problem is about.
### kindly check this👇
![review](https://user-images.githubusercontent.com/66992901/200529638-1140870e-5d2f-490b-b086-c279242d7a85.PNG)